pkgs/types/src/lib.zig

```diff
@@ -120,8 +109,8 @@ pub const BeamState = struct {
     justified_slots: JustifiedSlots,
 
     // a flat representation of the justifications map
-    justifications_roots: []Root,
-    justifications_validators: []u8,
+    justifications_roots: ssz.utils.List(Root, params.MAX_JUSTIFICATION_ROOTS),
+    justifications_validators: ssz.utils.Bitlist(params.MAX_JUSTIFICATION_VALIDATORS),
```

Comment:
check and correct the sizes and variable names:

https://github.com/leanEthereum/leanSpec/blob/main/docs/client/containers.md